SPECS/kernel-rt/kernel-rt.spec

```diff
@@ -175,6 +175,10 @@ make VERBOSE=1 KBUILD_BUILD_VERSION="1" KBUILD_BUILD_HOST="CBL-Mariner" ARCH=%{a
 # Compile perf, python3-perf
 make -C tools/perf PYTHON=%{python3} all
 
+%ifarch x86_64
+make -C tools turbostat cpupower
```

Comment:
Double checking here; I do not see a `%files` inclusion for these utils, so wondering are we even shipping them ?